.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,132 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: input
+    id: contact
+    attributes:
+      label: Contact Details
+      description: How can we get in touch with you if we need more info?
+      placeholder: ex. email@example.com
+    validations:
+      required: false
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Provide a clear and concise description of what the bug is.
+      placeholder: Tell us a description of the bug.
+    validations:
+      required: true
+  - type: textarea
+    id: step-to-reproduce
+    attributes:
+      label: Steps to reproduce
+      description: Provide details of how to reproduce the bug.
+      placeholder: ex. 1. Go to '...'
+    validations:
+      required: false
+  - type: textarea
+    id: expected-behavior
+    attributes:
+      label: Expected behavior
+      description: What you expected to happen?
+      placeholder: ex. I expected '...' to happen
+    validations:
+      required: false
+  - type: textarea
+    id: actual-behavior
+    attributes:
+      label: Actual behavior
+      description: What actually happened?
+      placeholder: ex. Instead '...' happened
+    validations:
+      required: false
+  - type: dropdown
+    id: operating-system
+    attributes:
+      label: Operating system
+      description: What operating system are you using?
+      options:
+        - Windows
+        - macOS
+        - Linux
+      default: 0
+    validations:
+      required: true
+  - type: dropdown
+    id: python-version
+    attributes:
+      label: Python version
+      description: Which version of Python are you using?
+      options:
+        - 3.12.0 
+        - 3.11.0 (Default)
+        - 3.10.0
+        - 3.9.0
+        - 3.8.0
+        - 3.7.0
+        - 3.6.0
+        - 3.5.0
+      default: 1
+    validations:
+      required: true
+  - type: dropdown
+    id: version
+    attributes:
+      label: Version
+      description: What version of PyCM are you using?
+      options:
+        - 4.0 (Default)
```

Comment:
Is it necessary to write that `(Default)`?

Reply:
I believe that we should include a pattern in the `version_check.py` script.

Reply:
Please take a look at https://github.com/sepandhaghighi/pycm/releases.
We have 43 releases!